wordpress/help.md

```diff
@@ -1,10 +1,11 @@
 # Description
 
-The [WordPress](https://wordpress.com/) plugin allows you to manage users on the WordPress platform
+The [WordPress](https://wordpress.com/) plugin allows you to manage users on the WordPress platform.  Only users can be deleted or suspended from the wordpress application.
```

Comment:
```suggestion
The [WordPress](https://wordpress.com/) plugin allows you to manage users on the WordPress platform. Only users can be deleted or suspended from the wordpress application.
```

Reply:
Double spaces after a period is keyboard muscle memory for me.  removed nonetheless